packages/cdk8s-cli/src/import/base.ts

```diff
@@ -69,6 +69,7 @@ export abstract class ImportBase {
 
           const opts: srcmak.Options = {
             entrypoint: fileName,
+            moduleKey: name,
```

Comment:
Can you explain?

Reply:
The `moduleKey` for `jsii-srcmak` is so consumers can choose how to set the package name in the `package.json` before running it through JSII (https://github.com/aws/jsii-srcmak/blob/master/src/compile.ts#L55)

This was previously just `generated` but there was a problem with multiple imports colliding with the same package name. There's a bunch more details in https://github.com/awslabs/cdk8s/issues/273 as well!

Reply:
Gotcha 👍